src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderNonAssociationDetails.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.util.Collection;
+
+@ApiModel(description = "Offender non-association details")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class OffenderNonAssociationDetails {
+    @ApiModelProperty(value = "Offender non-association details")
+    private Collection<OffenderNonAssociationDetail> details;
+}
```

Comment:
More musing than anything, what's the benefit of having the list of non-associations be wrapped in a details object? Only asking because I would've totally just returned `List<OffenderNonAssociationDetail>` from the resource and not considered doing a separate model.

Reply:
It is actually GDS guidelines.  I'll see if I can dig it out....

Reply:
https://www.gov.uk/guidance/gds-api-technical-and-data-standards#use-restful  (see section 'Use Json'